tests/test-utils.js

```diff
@@ -1,11 +1,13 @@
 if (global.DOMParser !== undefined) {
-  const parser = new DOMParser();
+  // We're in Firefox
   module.exports = {
     stringToDom(str) {
+      const parser = new DOMParser();
```

Comment:
nit: I don't think its necessary to instantiate a new `DOMParser` each time `stringToDom()` is called. We should be able to just reuse a single instance as before.